js-exercises/abbreviate-string/abbreviateString.js

```diff
@@ -0,0 +1,11 @@
+import { isString } from 'util';
+
+function abbreviateString(str) {
+  if (!isString(str) || !str.length) throw new Error(' Please use a valid string.');
```

Comment:
Check out different types of errors you can throw. In this case, `TypeError` seems appropriate to me. Not 100% sure though. 

Something like:
```js
throw new TypeError('Input must be a string');
```